db/models/news-models.js

```diff
@@ -86,10 +86,25 @@ const selectArticles = (order_by = "desc") => {
     });
 };
 
+const patchArticle = (inc_votes, article_id) => {
+  if (!inc_votes) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return db
+    .query(
+      `UPDATE articles SET votes = votes + ${inc_votes} WHERE article_id = $1 RETURNING *;`,
+      [article_id]
+    )
+    .then((result) => {
+      return result.rows;
```

Comment:
This will only be one article, so result.rows[0] would be more appropriate as a return value here to send back the article object instead of an array.